DigitalLearningSolutions.Web/Helpers/ViewDataHelper.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers
+{
+    public static class ViewDataHelper

```

Comment:
I think we could be a bit more specific here - how does `LayoutViewDataKeys` sound?
It's not worth making it more generic for View Data helper methods that might come up - they can go in another class.